### ISSUE TITLE
number of bam files for comparison 

### ISSUE THREAD

**Issue by ibseq** [cbg-ethz/cojac#5]
Hi all, 
is the comparison done only two bam files at the time? can we add a longer list?

thanks
ibseq

**Comment by DrYak** [comment]
Yes, you can: the `-a` / `--alignments` can take an arbitrarily large number off SAM/BAM/CRAM alignment files.

(Though in our own case, we mostly use it with the `-s` / `--samples` options, as we process our samples upstream with V-pipe).

Note that each sample will be processed sequentially (there's no built-in parallelism in the `cooc-mutbamscan` executable).

If your collection of samples if very large, you can split it into multiple jobs, and then simply merge the YAML or JSON output.

**Comment by ibseq** [comment]
vpipe is a variant caller, how do you generate the bam files. will any from the artic pipeline (illumina or ONT) will work? 
thanks 
ibseq
